### PR TITLE
send display data watch after starting viewer listener

### PR DESCRIPTION
**Pull request by santiagonoguez**
## Description
Fix to defect https://github.com/Rise-Vision/viewer/issues/566

## Motivation and Context
Templates based on rise-init running on Web Player sometimes miss the Content Sentinel display.json response. This happens because on web player the display.json request ( [here](https://github.com/Rise-Vision/common-template/blob/master/src/rise-display-data.js#L65) ) happens before the viewer listener starts ( [here](https://github.com/Rise-Vision/common-template/blob/master/src/rise-player-configuration.js#L191) ).

If Content Sentinel answers before the viewer listener for a template starts, that template won't receive the display.json file, and so it won't apply branding. I could verify this was indeed happening by turning on endpoint logging debug, and verifying the order of the messages ( [here](https://docs.google.com/spreadsheets/d/1a3o2MoDWO-pRaQIBCPs8DIzUeNUzgKJGzEHxdT1mi-Q/edit#gid=562062417) ). More extensive analysis in related [Trello card](https://trello.com/c/eXxV2pU1/6537-defect-566-template-branding-not-always-loading-in-web-player).

The solution is to send the display data watch for web player right after starting Viewer listener.

## How Has This Been Tested?
Describe in detail how you tested your changes. Include details of your testing environment and link(s) for reviewers to validate.

I created a custom version of common template, and verified in console that the display.json messages weren't lost in any case.

I used this template: https://apps.risevision.com/schedules/details/43cc8c2a-ffc7-47ab-a2c5-5efa40b1ee86?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f

and webplayer running here: https://widgets.risevision.com/viewer/?type=webplayer&id=FK8V3AC222ZR

 - [x] Common template has been tested on Player
 - [x] Common template has been tested on Chrome OS Player
 - [x] Common template has been tested on Web Player
 - [x] Common template has been tested in a Shared Schedule
 - [x] Common template has been tested in Apps Template Editor
 - [x] Common template has been tested in embedded template

## Release Plan:
To be released before Friday.
Simple rollback is enough to revert.

- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed.
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No need to notify support. No documentation updated ( please let me know if there is any to updated ).

@stulees @olegrise as this affects common-template's lifecycle, I'm including you on the review, thanks !

### CODE REVIEW

File: src/rise-player-configuration.js [hunk 1]
```diff
@@ -195,6 +195,10 @@ const RisePlayerConfiguration = (() => {
 
     if ( !RisePlayerConfiguration.Helpers.isTestEnvironment()) {
       _lockDownRisePlayerConfiguration();
+
+      if ( RisePlayerConfiguration.Helpers.isWebPlayer()) {
+        RisePlayerConfiguration.DisplayData.connectionHandler();
```
Comment (santiagonoguez) [inline src/rise-player-configuration.js:200]: A further improvement could be to expose and call directly _retrieveDisplayData() here - https://github.com/Rise-Vision/common-template/blob/master/src/rise-display-data.js#L22

And then avoiding the conditional for web player here: 
https://github.com/Rise-Vision/common-template/blob/master/src/rise-display-data.js#L41-L42

and the intention of this line with be more clear, but I wanted to keep this PR as simple as possible. thanks !

Comment (stulees) [reply src/rise-player-configuration.js:200]: @santiagoarriaga That sounds like a specific improvement to web player and perhaps not for all endpoint types as rise-display-data isn't only used for web player. I like the changes you've proposed in this PR.

**Review by stulees — COMMENTED** [review]
@santiagoarriaga Any opportunity for additional test coverage?

**Comment by santiagonoguez** [comment]
@stulees the code is executed in blocks that check for not being in test environment, so I don't think it can be easily tested, thoughts ?

The call to connection handler by web player itself is already covered here.
https://github.com/Rise-Vision/common-template/blob/master/test/unit/rise-display-data.test.js#L26

**Review by stulees — APPROVED** [review]
@santiagoarriaga No problem, thanks for clarifying.

**Review by olegrise — APPROVED** [review]

**Pull request merged**